src/cpp/session/modules/SessionRMarkdown.R

```diff
@@ -172,7 +172,47 @@
 
 .rs.addJsonRpcHandler("convert_to_yaml", function(input)
 {
-   yaml <- yaml::as.yaml(input)
+   # the yaml package doesn't treat string values kindly if they're surrounded
+   # by backticks, so we will need to replace those with UUIDs we can sub out
+   # later
+   exprs <- list()
+   tick_sub <- function(x) 
+   {
+      lapply(x, function(val) 
+      {
+         if (is.list(val)) 
+         {
+            # if it's a list, recurse
+            tick_sub(val)
+         }
+         else if (is.character(val)) 
```

Comment:
I'm not sure (maybe in string arrays of Pandoc args?). To be safe though I've added a guard: https://github.com/rstudio/rstudio/pull/968/commits/f5dd13602f27bf7cbb6e8223671d73ae217cb6e3